uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/MediaPopupElementView.qml

```diff
@@ -0,0 +1,169 @@
+        }
+        Label {
+            text: controller.description
+            visible: controller.description !== ""
+            wrapMode: Text.WordWrap
+
+            Layout.fillWidth: true
+            Layout.leftMargin: mediaMargin
+            Layout.rightMargin: mediaMargin
+        }
+    }
+
+    MenuSeparator {
+        Layout.fillWidth: true
+        Layout.leftMargin: mediaMargin
+        Layout.rightMargin: mediaMargin
+    }
+
+    ListView {
+        orientation: ListView.Horizontal
+        clip: true
+        focus: true
+        height: 170
+        spacing: mediaMargin
+        model: controller.popupMediaItems
+
+        Layout.fillWidth: true
+        Layout.leftMargin: mediaMargin
+        Layout.rightMargin: mediaMargin
+
+        delegate: Item {
+            height: 170
+            width: 220
+
+            Image {
+                id: imageMediaPreview
+                source: model.sourceUrl
+                fillMode: Image.PreserveAspectCrop
+                asynchronous: true
+                cache: true
+
+                anchors.fill: parent
+                Layout.leftMargin: mediaMargin
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        // saving this as a placeholder. Should fire an event the user could take otherwise
+                        // we define our own default behavior to open the image in full screen mode.
+                        // Qt.openUrlExternally(model.linkUrl);
+                    }
+                }
+            }
+
+            // Overlay text
+            Rectangle {
+                visible: model.title !== ""
+                height: overlayTextLayout.height
+                color: "white"
+                opacity: 0.7
+
+                anchors {
+                    bottom: parent.bottom
+                    left: parent.left
+                    right: parent.right
+                }
+
+                ColumnLayout {
+                    id: overlayTextLayout
+                    height: childrenRect.height
+                    clip: true
+                    focus: true
+                    spacing: layoutSpacing
+
+                    anchors.bottom: parent.bottom
+
+                    Label {
+                        visible: model.title !== ""
+                        text: model.title
+                        font.weight: Font.Bold
+                        color: "black"
+                        elide: Text.ElideRight
+
+                        Layout.alignment: Qt.AlignHCenter
+                        Layout.maximumWidth: imageMediaPreview.width - imageTextMargin
+                        Layout.leftMargin: imageTextMargin
+                        Layout.rightMargin: imageTextMargin
+                    }
+
+                    Label {
+                        visible: model.caption !== ""
+                        text: model.caption
+                        color: "black"
```

Comment:
Same questions as Lucas about hard-coded color choices and how those will play with dark mode support.

Reply:
@JamesMBallard Does the following explanation suffice? https://github.com/Esri/arcgis-maps-sdk-toolkit-qt/pull/648#issuecomment-2450806207

Reply:
Yes it does, thanks.